smarty/sampler.py

```diff
@@ -129,41 +144,16 @@ def __init__(self, molecules, basetypes_filename, initialtypes_filename, decorat
                 self.atomtypes = [(smarts, typename)] + self.atomtypes
```

Comment:
Specifically, you asked Caitlin to specify all possible new base types. Since any SMARTS pattern could be a base type, and there are an infinite number of possible SMARTS patterns, there are an infinite number of possible base types. What is it you're trying to accomplish by knowing and/or classifying all possible base types? I think we're not understanding why you want this list or why you want to classify them, so you're going to need to explain.

Also, "That should fix this" is unclear to me -- what's the "that" and the "this" that are referred to?